docs: Conceptual Docs for Api Gateways, The Gateway Api & Envoy Proxy (#5739)
envoyproxy/gateway: site/content/en/latest/concepts/introduction/envoy_proxy.md

---
title: "Envoy Proxy"
---

## Overview

A proxy server acts as an intermediary between a client (like a web browser) and another server (such as a website). When the client makes a request, the proxy forwards it to the destination server, receives the response, and then sends it back to the client.

Proxies are commonly used to enhance security, manage traffic, anonymize user activity, or optimize performance through features like caching and load balancing. In cloud environments, they often handle critical tasks such as request routing, TLS termination, authentication, and traffic shaping.

> Envoy Proxy is a high-performance, open source proxy built for cloud-native applications. Originally developed at Lyft and now a graduated project in the Cloud Native Computing Foundation (CNCF), Envoy supports both edge proxy and service proxy use cases—routing traffic at the boundary of a system or between internal services.

Reviewer: Consider putting this paragraph first in the this `Overview` section

> Envoy Proxy is a high-performance, open source proxy built for cloud-native applications. Originally developed at Lyft and now a graduated project in the Cloud Native Computing Foundation (CNCF), Envoy supports both edge proxy and service proxy use cases—routing traffic at the boundary of a system or between internal services.

Reviewer: ```suggestion
Envoy Proxy is a high-performance, open-source proxy designed for cloud-native applications. Originally developed at Lyft and now a graduated project in the Cloud Native Computing Foundation (CNCF), Envoy supports use cases for edge and service proxies, routing traffic at the system’s boundary or between internal services.
```
